### DISCUSSION TITLE
change character modchart not working

### DISCUSSION THREAD

**Discussion by Tikoulla** [KadeArchive/Kade-Engine#2836] [Q&A]
i'm trying to change the character in a curstep but it doesn't work
here is the code:
function start(song) -- arguments, the song name

end

function update(elapsed) -- arguments, how long it took to complete a frame

function beatHit(beat) -- arguments, the current beat of the song

end

function stepHit(step) -- arguments, the current step of the song (4 steps are in a beat)
    if curstep == 1152 then
    changeDadCharacter('meri-forced')
end
    if curstep == 1664 then
    changeDadCharacter('meri-mad')
end
    if curstep == 1920 then
    changeDadCharacter('meri-forced')
    end
end

end

**Comment by ghost** [comment]
That is not the right line of code lol. It's supposed to be this.
```
dad:changeCharacter("mom",30,30)
```
I straight up copied this from the Modchart wiki.
Change "mom" to the character you want, and change the numbers to the changed x and y position.
I believe this is the script you want?
```
if curstep == 1152 then
    dad:changeCharacter("meri-forced",0,0)
end
if curstep == 1664 then
    dad:changeCharacter("meri-mad",0,0)
end
if curstep == 1920 then
    dad:changeCharacter("meri-forced",0,0)
end
```
Hope this helps!

**Comment by Tikoulla** [comment]
it's not working :(
